tests/issues/issue-996.php

```diff
@@ -111,7 +109,6 @@
             'JavaScript' => true,
             'VBScript' => false,
             'JavaApplets' => false,
-            'isSyndicationReader' => false,
```

Comment:
Why is this property being removed from tests? Is it one of the deprecated properties we're removing as part of #1433?

Are the tests modified in any other way? There are a lot of files to expand here to see what was modified, especially if the only change is the removal of this property.

Reply:
>  Is it one of the deprecated properties we're removing as part of #1433?

Yes it is.

> Are the tests modified in any other way?

There are some test cases where I added a seperate tests for the lite mode. (These are the test files with added lines)

Reply:
>> Is it one of the deprecated properties we're removing as part of #1433?

Removing this property was a mistake. I looked not properly enough at the deprecated list.